services/Google Ads.json

```diff
@@ -35,6 +35,10 @@
       "fetch": "https://developers.google.com/adwords/api/docs/terms",
       "select": ".devsite-article-body",
       "remove": "style"
-    }
+    },
+    "Ad Publishing Policy": {
```

Comment:
Hey,

Ad Publishing Policy is not an available type of document. 
You should add it here : `OpenTermsArchive/blob/master/src/app/types.json`

Ask for any question you need 👌

Reply:
Done!

Reply:
As soon as [this PR](https://github.com/ambanum/OpenTermsArchive/pull/264) is corrected I can validate this one ;)